DI-1744/get_xlsx_paths.py

```
@@ -0,0 +1,243 @@
+        List of sample IDs to search for in the project.
+
+    Returns
+    -------
+    records: list
+        A list of dictionaries containing sample ID, project ID, and file name.
+    """
+    records = []
+    # pattern = rf'^\d+-({"|".join(sample_ids)})-[\w-]+_R\d+\.\d_(?:SNV|CNV)_1\.xlsx$'
+    pattern = rf'.*({"|".join(sample_ids)}).*xlsx'
+    try:
+            matching_files = dxpy.find_data_objects(
```

Pylint (3.3.7)

[warning] Bad indentation. Found 12 spaces, expected 8

(W0311)